### ISSUE TITLE
Print snapshot/backup statistics

### ISSUE THREAD

**Issue by digint** [digint/btrbk#55]
Add command to btrbk which prints statistics on number of snapshots and backups (see also conversation in issue #54)

Something like this:
```
# btrbk stats

/mnt/btr_pool/rootfs  (42 snapshots)
^-- /mnt/btr_backup/  (130 backups)

/mnt/btr_pool/data  (42 snapshots, up-to-date)
^-- /mnt/btr_backup/  (213 backups, up-to-date)
^-- backup1.mydomain.com:/mnt/btr_backup/mysourcehost/  (312 backups, up-to-date)
^-- backup2.mydomain.com:/mnt/btr_backup/mysourcehost/  (311 backups)
^-- backup3.mydomain.com:/mnt/btr_backup/mysourcehost/  (host unreachable)
```

**Comment by GrahamCobb** [comment]
That looks good! :smiley: 

One useful addition for cases when the snapshot is not up to date would be to show the date (and time - I do hourly snapshots of some subvols) of the most recent snapshot.  That seems relatively hard to find.

**Comment by digint** [comment]
Implemented in: e7ff20114c28c88a7082465ee3912649e736f2ac

The output looks like this:
```
# ./btrbk stats beta
--------------------------------------------------------------------------------
Statistics (btrbk command line client, version 0.21.1-dev)

    Date:   Fri Jan 15 02:19:48 2016
    Config: /etc/btrbk/btrbk.conf
    Filter: group=beta
--------------------------------------------------------------------------------
/mnt/btr_system/root_gentoo        28 snapshots
^-- /mnt/btr_ext/_btrbk            19 backups (5 orphaned)
^-- /mnt/btr_backup/beta.tty0.ch  103 backups (75 orphaned)
/mnt/btr_system/kvm                 1 snapshots
^-- /mnt/btr_backup/beta.tty0.ch   66 backups (65 orphaned)
/mnt/btr_data/home                 30 snapshots
^-- /mnt/btr_backup/beta.tty0.ch  101 backups (73 orphaned)
/mnt/btr_ext/data                  29 snapshots
^-- /mnt/btr_backup/beta.tty0.ch  103 backups (75 orphaned)
/mnt/btr_boot/boot                  1 snapshots
^-- /mnt/btr_ext/_btrbk             2 backups (1 orphaned)
^-- /mnt/btr_backup/beta.tty0.ch    4 backups (3 orphaned)

Total (group=beta):
 99 snapshots
398 backups (297 orphaned)
```

Comments welcome!

**Comment by digint** [comment]
Implemented in btrbk v0.22.0
